lib/utils/token/getEntitlements.test.ts

```diff
@@ -30,6 +30,28 @@ const mockEntitlementResponse = {
   },
 };
 
+const expectedResponse = {
+  orgCode: "org_0195ac80a14e",
+  plans: [
+    {
+      key: "pro_plan",
+      subscribedOn: "2025-06-01T12:00:00Z",
+    },
+  ],
+  entitlements: [
+    {
+      id: "entitlement_0195ac80a14e8d71f42b98e75d3c61ad",
+      fixedCharge: 35,
+      priceName: "Pro gym",
+      unitAmount: 1,
+      featureCode: "base_price",
```

Comment:
@DanielRivers can we make this featureKey as that's what we have in the UI, i'll check the API spec and make sure it reflects this